src/UI/Art/SpriteArtist/DamageChartView.java

```diff
@@ -0,0 +1,220 @@
+  private SlidingValue viewX = new SlidingValue(0);
+  private SlidingValue viewY = new SlidingValue(0);
+  private SpriteCursor spriteCursor = new SpriteCursor(0, 0, SpriteLibrary.baseSpriteSize, SpriteLibrary.baseSpriteSize, UIUtils.getCOColors()[0]);
+
+  public DamageChartView(DamageChartController control)
+  {
+    this.control = control;
+    unitArray = new BufferedImage[control.units.length];
+
+    // Just pull one sprite to start with. Loading everything at once might take a while.
+    unitArray[0] = SpriteLibrary.getMapUnitSpriteSet(control.units[0].name, control.shooters, control.shooterColor).sprites[0].getFrame(0);
+
+    unitSizePx = unitArray[0].getHeight(); // Units are square.
+    unitBuffer = unitSizePx / 3; // Space between options in the grid.
+    unitSpacingH = unitSizePx + unitBuffer;
+    unitSpacingV = unitSizePx;
+    gridWidth = control.units.length * unitSpacingH - unitBuffer;
+    gridHeight = control.units.length * unitSpacingV;
+  }
+
+  @Override
+  public void render(Graphics g)
+  {
+    int indexShooter = control.getSelectedShooter();
+    int indexTarget = control.getSelectedTarget();
+
+    // Get the draw space
+    int drawScale = SpriteOptions.getDrawScale();
+    Dimension dimensions = SpriteOptions.getScreenDimensions();
+    int myWidth = dimensions.width / drawScale;
+    int myHeight = dimensions.height / drawScale;
+
+    // Don't move if we don't have to
+    int startX = viewX.geti();
+    int startY = viewY.geti();
+
+    // Margins are the amount of screenspace we'd like to be between the cursor and the edge of the screen
+    int marginFrac = 5;
+    int marginX = myWidth/marginFrac;
+    int marginY = myHeight/marginFrac;
+    // virtual drawspace coordinate of the selection, relative to the top-left of the grid
+    int selX = indexShooter*unitSpacingH;
+    int selY = indexTarget*unitSpacingV;
+
+    startX = findViewDestination(selX, gridWidth,  unitSizePx, unitBuffer, startX, myWidth  - unitSpacingH, marginX);
+    startY = findViewDestination(selY, gridHeight, unitSizePx, unitBuffer, startY, myHeight - unitSpacingV, marginY);
+
+    viewX.set(startX, false);
+    viewY.set(startY, false);
+
+    // Render it in true size, and then we'll scale it to window size as we draw it.
+    BufferedImage image = SpriteLibrary.createTransparentSprite(myWidth, myHeight);
+    Graphics myG = image.getGraphics();
+
+    // Draw a highlight on the current cursor row/column
+    myG.setColor(SpriteUIUtils.MENUFRAMECOLOR);
+    myG.fillRect(selX + unitSpacingH - viewX.geti(), 0, unitSizePx, myHeight);
+    myG.fillRect(0, selY + unitSpacingV - viewY.geti(), myWidth, unitSizePx);
+
+    // Draw the top unit "labels"
+    for(int i = 0; i < control.units.length; ++i)
+    {
+      int yOff = 0;
+      int xOff = unitSpacingH * (i+1) - viewX.geti();
+      if( (xOff > myWidth ) || (xOff < unitSpacingH) )
+        continue;
+      drawTargetAt(myG, i, xOff, yOff);
+    }
+
+    // t = target, s = shooter
```

Comment:
Would be good to delineate that left is shooter and top is target. Maybe just by using the current player's color on the left and some other color on the top.
This creates the potential issue that if you choose a different player color along the top, it could give the impression that this table takes current CO choices into account, when the values are bland. Maybe a header above the table to specify that these are default values? But that's less useful - players will want values relevant to their current CO by default.